z00_lmod.sh

```diff
@@ -5,71 +5,72 @@
 #
 ########################################################################
 
-if [ -z "${USER_IS_ROOT:-}" ]; then
+# Skip Lmod initialization if USER_IS_ROOT or LMOD_DISABLE is set
 
-  if [ -z "${MODULEPATH_ROOT:-}" ]; then
-    export USER=${USER-${LOGNAME}}  # make sure $USER is set
-    export LMOD_sys=`uname`
+([ -n "$USER_IS_ROOT" ] || [ -n "$LMOD_DISABLE" ]) && exit
```

Comment:
@johnnydevaprasad : OK, after reading this, it's a pass from me:
https://rgeissert.blogspot.co.uk/2013/02/a-bashism-week-short-circuiting-tests.html